backend/src/database/entities/availability.entity.ts

```diff
@@ -19,12 +22,15 @@ export class AvailabilityEntity {
   @Column({
     name: 'availability_type',
     type: 'enum',
-    enum: AvailabilityType,
+    enum: AvailabilityTypeName,
```

Comment:
I'd again prefer to keep this as AvailabilityType, ie. all caps, underscores on the database.